src/Exception/MissingDependencyException.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
- * @copyright Copyright (c) 2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @copyright Copyright (c) 2016-2017 Zend Technologies USA Inc. (http://www.zend.com)
```

Comment:
For some reason, this one kept getting flagged until I changed the date range. I have no idea why, and I didn't want to waste more time on it.